Aspire.sln

```diff
@@ -3915,7 +3915,7 @@ Global
 		{39AD3DBA-6B79-44FC-9637-232544D0CA46} = {27381127-6C45-4B4C-8F18-41FF48DFE4B2}
 		{0C97F423-DAAE-4B63-B1B1-CB85E2840B98} = {27381127-6C45-4B4C-8F18-41FF48DFE4B2}
 		{23075FC1-D893-434D-9A20-02870B96ABA7} = {C424395C-1235-41A4-BF55-07880A04368C}
-		{F371C3EE-8AB3-4261-A5F2-9A1FF77ADC19} = {C424395C-1235-41A4-BF55-07880A04368C}
+		{F371C3EE-8AB3-4261-A5F2-9A1FF77ADC19} = {4981B3A5-4AFD-4191-BF7D-8692D9783D60}
```

Comment:
Is this intentional?

Reply:
Yes, moved Aspire.Components.Common.Tests.csproj to the "tests" v-folder because it is used across all tests.
![image](https://github.com/user-attachments/assets/f9b1bd81-c345-4957-8f3e-b6715c5316f9)

I'm of an option the project should be renamed to not imply it is a project that contains tests (e.g., to "Aspire.TestsUtilities.csproj"), as its current names implies just that.

Reply:
Fixed in #8279